pipelines/qc/Snakefile

```diff
@@ -1,36 +1,38 @@
 import os
 import numpy as np
 import pandas as pd
+import re
 
-sampleset = os.path.split(os.getcwd())[1]
 if os.path.isfile("configuration.yml"):
   os.remove("configuration.yml")
 
 # read_fofn
-fofn_path = os.path.join(config["vector_ops"], "tracking", sampleset, "agam.fofn.tsv")
+pat = re.compile("^.+(AG1000G-\D{2}(-\D)?)/agam.fofn.tsv$")
```

Comment:
This will break on VOBS sample sets. Why we cannot just use `fofn_path.split('/')[-2]`?